packages/pnpmfile/src/requireHooks.ts

```diff
@@ -92,3 +105,12 @@ function createReadPackageHookContext (calledFrom: string, prefix: string, hook:
     }),
   }
 }
+
+function createPreResolutionHookLogger (prefix: string): PreResolutionHookLogger {
+  const hook = 'preResolution'
+
+  return {
+    info: (message: string) => hookLogger.info({ message, prefix, hook } as any), // eslint-disable-line
+    warn: (message: string) => hookLogger.warn({ message, prefix, hook } as any), // eslint-disable-line
```

Comment:
❓ @zkochan Is it expected that on the `hookLogger` it won't show the yellow warning batch when you call `.warn()`?

Reply:
Does it show the message at all?

Reply:
Yes it does, but it's not shown as a warning.

Reply:
I am not sure why. It is probably how it is implemented in the default-reporter package.